CHANGELOG.rst

```diff
@@ -5,6 +5,13 @@ Changelog
 dev-version
 ===========
 
+Breaking Changes
```

Comment:
Should this be added to a `v4.0.0 [yyyy.mm.dd]` entry?

Reply:
Yeah, I was thinking of doing that in a separate PR, but might as well do it here.